cockatrice/src/dlg_settings.cpp

```diff
@@ -246,6 +249,7 @@ void GeneralSettingsPage::retranslateUi()
     highQualityURLLabel.setText(tr("Custom Card Download URL:"));
     highQualityURLLinkLabel.setText(QString("<a href='%1'>%2</a>").arg(LINKING_FAQ_URL).arg(tr("Linking FAQ")));
     clearDownloadedPicsButton.setText(tr("Reset/Clear Downloaded Pictures"));
+    updateNotificationCheckBox.setText(tr("Notify when client updates are available"));
```

Comment:
This is a bit misleading I think.
The client won't inform you in general about a new update.
He will just inform you if you connect to a server running a newer "version" (or just if there are new features? no important bugfixes included?)

brainstorming:
`Allow a server you connect to, to inform you that he runs a newer version than your client`
`Enable notifications if a server supports more features (new version/update available)`
...